NEWS.md

```diff
@@ -5,7 +5,6 @@ This version features an overhaul of the coercion system to make it
 more consistent and easier to implement. See the _Breaking changes_
 and _Type system_ sections for details.
 
-
```

Comment:
Can we keep the formatting for pure-text viewing/editing?

Reply:
I just messed up the merge

Reply:
I will, however, remove the empty line at the start of the doc, since that messes up the usethis release helper